boards/cc2538dk/include/periph_conf.h

```diff
@@ -22,79 +22,14 @@
 #include "cpu.h"
 #include "periph_cpu.h"
 
+#include "cfg_clk_default.h"
+#include "cfg_rtt_default.h"
+#include "cfg_timer_default.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-/**
- * @name    Clock system configuration
- * @{
- */
-/*
- * 0: use internal 32KHz RCOSC
- * 1: use external 32KHz XOSC
- */
-#ifndef SYS_CTRL_OSC32K_USE_XTAL
-#define SYS_CTRL_OSC32K_USE_XTAL        (1)
-#endif
-/*
- * 0: use internal 16MHz RCOSC
- * 1: use external 32MHz XOSC, required for RF operation
- */
-#ifndef SYS_CTRL_OSC_USE_XTAL
-#define SYS_CTRL_OSC_USE_XTAL           (1)
-#endif
-
-#if SYS_CTRL_OSC_USE_XTAL
-#define CLOCK_OSC           (XOSC32M_FREQ)
-#else
-#define CLOCK_OSC           (RCOSC16M_FREQ)
-#endif
-
-#if SYS_CTRL_OSC32K_USE_XTAL
-#define CLOCK_OSC32K        (XOSC32K_FREQ)    /* XCOSC frequency */
-#else
-#define CLOCK_OSC32K        (RCOSC32K_FREQ)    /* XCOSC frequency */
-#endif
-
-/* System clock frequency 32MHz */
-#define CLOCK_CORECLOCK     (CLOCK_OSC)
-/* I/O clock rate setting 16MHz */
-#define CLOCK_IO            (CLOCK_OSC / 2)
-/** @} */
-
-/**
- * @name    Timer configuration
- *
- * General purpose timers (GPT[0-3]) are configured consecutively and in order
- * (without gaps) starting from GPT0, i.e. if multiple timers are enabled.
- *
- * @{
- */
-static const timer_conf_t timer_config[] = {
-    {
-        .chn = 2,
-        .cfg = GPTMCFG_16_BIT_TIMER, /* required for XTIMER */
-    },
-    {
-        .chn = 1,
-        .cfg = GPTMCFG_32_BIT_TIMER,
-    },
-    {
-        .chn = 2,
-        .cfg = GPTMCFG_16_BIT_TIMER,
-    },
-    {
-        .chn = 1,
-        .cfg = GPTMCFG_32_BIT_TIMER,
-    },
-};
```

Comment:
This wasn't added to the common board config.

Reply:
There was a file under `common/remote` `cfg_timer_default`, which had the same configuration, I just moved that to `common/cc2538` or Am I missing something?

`...common/remote/include/cfg_timer_default.h → ...common/cc2538/include/cfg_timer_default.h `

Reply:
Ah didn't see that, I only looked at the diff on Github.